modules/xcode/xcode_project.lua

```diff
@@ -15,6 +15,39 @@
 	local fileconfig = p.fileconfig
 	local tree = p.tree
 
+--
+-- Checks if a node must be excluded completely from a target or not. It will
+-- return true only if the node has the "ExcludeFromBuild" flag in all the
+-- configurations.
+--
+-- @param node
+--    The node to check.
+-- @param prj
+--    The project being generated.
+-- @returns
+--    A boolean, telling whether the node must be excluded from its target or not.
+--
+	function xcode.mustExcludeFromTarget(node, prj)
+		if not node.configs then
+			return false
+		end
+
+		local value
+		for cfg in premake.project.eachconfig(prj) do
+			local filecfg = premake.fileconfig.getconfig(node, cfg)
+			if filecfg then
+				local newValue = not not filecfg.flags.ExcludeFromBuild

```

Comment:
not not not! not not?

Reply:
It's a hack to convert it into a boolean. If it's nil, not not nil == false ;)

Reply:
Got it 👍